### PR TITLE
OSDOCS#10085 reorganizing AWS installation requirements (IPI)

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10085

Link to docs preview:
[Installation methods](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/preparing-to-install-on-aws)

[Preparing to install a cluster (IPI)](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/ipi-aws-preparing-to-install)

[Installing a cluster](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-default)
[Installing a cluster in China](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-china.html)
[Installing a cluster with customizations](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-customizations.html)
[Installing a cluster in a government region](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-government-region.html)
[Installing a cluster on AWS Local Zones](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-localzone.html)
[Installing a cluster with network customizations](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-network-customizations.html)
[Installing a private cluster](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-private.html)
[Installing a cluster into a secret region](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-secret-region.html)
[Installing a cluster on an existing VPC](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-vpc.html)
[Installing a cluster on AWS Wavelengths](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-wavelength-zone.html)
[Installing a cluster on a restricted network](https://75476--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-restricted-networks-aws-installer-provisioned.html)

No QE required as these changes are only structural.

- Move "prerequisites" from global [preparing-to-install-on-aws.adoc](https://github.com/openshift/openshift-docs/pull/75476/files#diff-c1d5a01bc33e8d8e3a8f79665b3ce47987156900bf6c703b4e536d9b4407f510) file into IPI-specific [installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc](https://github.com/openshift/openshift-docs/pull/75476/files#diff-40ed035cae926c45bcf01eb94367ca8b6e4d4c673bb72b83fc14beeb4d6fd250) file
- Change heading of [preparing-to-install-on-aws.adoc](https://github.com/openshift/openshift-docs/pull/75476/files#diff-c1d5a01bc33e8d8e3a8f79665b3ce47987156900bf6c703b4e536d9b4407f510) to "Installation methods" (keeping filename the same to avoid any 404/redirect issues)
- Move SSH, entitlements, CLI, OC and telemetry modules out of installation assemblies and into [installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc](https://github.com/openshift/openshift-docs/pull/75476/files#diff-40ed035cae926c45bcf01eb94367ca8b6e4d4c673bb72b83fc14beeb4d6fd250) file

### CODE REVIEW

File: installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
```
Comment (JoeAldinger) [inline installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc:39]: The move looks good to me.

**Comment by JoeAldinger** [comment]
Make sure to squash before you merge. The merge messages and the diffs helped for review, thank you.

**Review by JoeAldinger — COMMENTED** [review]
A question and small comment, but this lgtm. I pulled it down to double check 😄 .

File: installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
++
+[NOTE]
+====
+If you are installing in a disconnected environment, install `oc` to the mirror host.
+====
+* Generating an SSH key pair. You can use this key pair to authenticate into the {product-title} cluster's nodes after it is deployed.
+
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, xref:../../../installing/installing_aws/ipi/installing-aws-customizations.adoc#manually-create-iam_installing-aws-customizations[manually creating long-term credentials for AWS] or xref:../../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-with-short-term-creds_installing-aws-customizations[configuring an AWS cluster to use short-term credentials] with Amazon Web Services Security Token Service (AWS STS).
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
```
[the remark this answers is not in the document]
Comment (bscott-rh) [reply installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc:31]: Definitely. I started fixing all of the comments and the PR immediately became XL+. will resolve later in the reorg

**Pull request merged**